packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -187,17 +195,17 @@ class DraggableScrollableNotification extends Notification with ViewportNotifica
     required this.maxExtent,
     required this.initialExtent,
     required this.context,
-  }) : assert(extent != null),
-       assert(initialExtent != null),
-       assert(minExtent != null),
-       assert(maxExtent != null),
-       assert(0.0 <= minExtent),
-       assert(maxExtent <= 1.0),
-       assert(minExtent <= extent),
-       assert(minExtent <= initialExtent),
-       assert(extent <= maxExtent),
-       assert(initialExtent <= maxExtent),
-       assert(context != null);
+  })  : assert(extent != null),
```

Comment:
My IDE polluted the PR with all this garbage-I thought I was using the canonical Flutter fmt, if not how do I use it so I don't pollute commits? Or was this file originally committed with non-canonical formatting?